nginx.conf

```diff
@@ -1,34 +1,33 @@
-user  www-data;
-worker_processes  1;
+user             www-data;
+worker_processes 1;
 
-error_log  /var/log/nginx/error.log warn;
-pid        /var/run/nginx.pid;
+error_log /var/log/nginx/error.log warn;
+pid       /var/run/nginx.pid;
 
 events {
-    worker_connections  1024;
+    worker_connections 1024;
 }
 
 http {
-
-    upstream backend {
-      server app-server:9000;
+    upstream php-handler {
+        server nextcloud-server:9000;
```

Comment:
Using a different key here than the current `app-server` in `docker-compose` will result in things working anymore.

Reply:
Corrected